tests/test_copy_novaseqx_flow_cell.py

```diff
@@ -0,0 +1,180 @@
+from pathlib import Path
+import pytest
+from typing import Set
+from cg.cli.demultiplex.copy_novaseqx_data import get_latest_analysis_directory
+
+from cg.cli.demultiplex.demux import (
+    copy_flow_cell_analysis_data,
+    is_ready_for_post_processing,
+)
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+@pytest.fixture
+def latest_analysis_version() -> str:
+    return "2"
+
+
+@pytest.fixture
+def flow_cell_name() -> str:
+    return "20230427_LH00188_0001_B223YYCLT3"
+
+
+@pytest.fixture
+def flow_cell_directory(tmp_path: Path, flow_cell_name: str) -> Path:
+    return tmp_path / flow_cell_name
+
+
+@pytest.fixture
+def demultiplexed_runs(tmp_path: Path) -> Path:
```

Comment:
name and scope in decorator and docstring that it is a temporary path to not confuse it with the one we have in fixtures

Reply:
Renaming the fixture with the name parameter seems to be an anti-pattern so I will not change this. Using the default scope (per test) for simple string fixtures is fine since it does not take any time to create them. 

I do think the fixtures need better names though, fixing that